Resources/Locale/en-US/_CM14/cm-chatsan.ftl

```diff
@@ -0,0 +1,71 @@
+﻿Universal:
+deltard
+retard = ""
+tard = ""
+
+Humanoid:
+admin = high command
+eorg = warcrimes
+
+
+Xeno:
+sniper = long spitter
+flare = glow stick
+synth = fake hosts
+synthetic = fake hosts
+synths = fake hosts
+synthetics = fake hosts
+human = tall host
+humans = tall hosts
+lizard = scaled host
+lizards = scaled hosts
+reptilian = scaled host
+reptilians = scaled hosts
+dwarf = short host
+dwarves = short hosts
+slime = slime host
+slimes = slime hosts
+diona = plant host
+dionas = plant hosts
+moth = winged host
+moths = winged hosts
+mothes = winged hosts
+spider = webspitter
+spiders = webspitters
+arachnid = webspitter
+arachnids = webspitters
+dropship = metal bird
+dropships = metal birds
+alamo = metal bird
+normandy = metal bird
+almayer = metal hive
+admin = grandmother
+ob = skyfire
+orbital = skyfire
+orbital bomb = skyfire
+orbital bombardment = skyfire
+engie = tall builder
+engineer = tall builder
+comtech = tall builder
+combat technitian = tall builder
+corpsman = tall healer
+medic = tall healer
+doc = tall healer
+doctor = tall healer
+squad leader = tall leader
+sl = tall leader
+commanding officer = tall queen
+co = tall queen
+xo = young tall queen
+executive officer = young tall queen
+so = young tall queen
```

Comment:
wouldn't this pose issues with normal speech? like saying "so what?"

Reply:
Thanks